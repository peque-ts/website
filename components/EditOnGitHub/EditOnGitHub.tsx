import React from 'react';

interface Props {
  link: string;
}

export const EditOnGitHub: React.VFC<Props> = ({ link }) => (
  <div className="mt-8 border-t border-t-secondary-700 pt-8 text-right">
    <a href={link} target="_blank" className="link text-sm" rel="noreferrer">
      Edit this page on GitHub
    </a>
  </div>
);
