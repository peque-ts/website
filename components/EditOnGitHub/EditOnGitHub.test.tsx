import { render } from '@testing-library/react';

import { EditOnGitHub } from './EditOnGitHub';

describe('EditOnGitHub', () => {
  it('should render', () => {
    const { asFragment } = render(
      <EditOnGitHub link="https://github.com/pequehq/website/blob/main/docs/framework/architecture.md" />,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
