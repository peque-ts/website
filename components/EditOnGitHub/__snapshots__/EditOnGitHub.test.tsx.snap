// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditOnGitHub should render 1`] = `
<DocumentFragment>
  <div
    class="pt-8 mt-8 border-t border-t-secondary-700 text-right"
  >
    <a
      class="link text-sm"
      href="https://github.com/pequehq/website/blob/main/docs/framework/architecture.md"
      rel="noreferrer"
      target="_blank"
    >
      Edit this page on GitHub
    </a>
  </div>
</DocumentFragment>
`;
